package.json

```diff
@@ -11,7 +11,9 @@
   "scripts": {
     "build": "ember build",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "ember test",
+    "build-storybook": "ember build && build-storybook -p 9001 -s dist",
+    "storybook": "ember serve & start-storybook -p 9001 -s dist"
```

Comment:
adding `--environment=test` after both `ember build` and `ember serve` will fix the `Ember Error: Assertion Failed: You cannot make a new Ember.Application using a root element that is an ancestor of an existing Ember.Application` https://github.com/storybooks/storybook/issues/5134